go/provisioner/local/config.go

```diff
@@ -476,6 +518,19 @@ func (p *localProvisioner) GeneratePgBackRestConfigs() error {
 			})
 		}
 
+		// Create per-pooler spool and lock directories inside backup repo
```

Comment:
@cuongdo  could you double check this makes sense? After thinking a bit more about this, I think the spool and the lock should be per pooler. But I'm not 100% sure